.eslintrc.json

```diff
@@ -9,7 +9,7 @@
   ],
   "rules": {
     "import/order": [
-      "error",
+      "warn",
```

Comment:
If you do this, will it still reorder everything with the lint fix?

Reply:
Why are you changing this, though?

Reply:
It won't fix, I did it cause the build was failing and I didn't want to change all the files.

If you prefer I can run a lint fix, but this will cause changes in many files (like 86)